### DISCUSSION TITLE
Build failed on "Ubuntu 18.04.6 LTS (Bionic Beaver)"

### DISCUSSION THREAD

**Discussion by Duality142857** [BoomingTech/Piccolo#312] [General]
**Describe the bug | Bug描述**
I tried to build on ubuntu18 with build_linux.sh but failed due to undefined references.

**Steps to reproduce | 如何复现**
1. ./build_linux.sh release

**Expected behavior | 预期行为**
    
**Actual behavior | 实际行为**
A clear and concise description of what actually happened.

**Screenshots/Logs | 截图/日志**
If applicable, add screenshots and/or a video to help explain your problem.

../engine/source/meta_parser/3rd_party/LLVM/bin/Linux/libclang.so.12: undefined reference to `tigetnum@NCURSES6_TINFO_5.0.19991023'
../engine/source/meta_parser/3rd_party/LLVM/bin/Linux/libclang.so.12: undefined reference to `log2@GLIBC_2.29'
../engine/source/meta_parser/3rd_party/LLVM/bin/Linux/libclang.so.12: undefined reference to `del_curterm@NCURSES6_TINFO_5.0.19991023'
../engine/source/meta_parser/3rd_party/LLVM/bin/Linux/libclang.so.12: undefined reference to `setupterm@NCURSES6_TINFO_5.0.19991023'
../engine/source/meta_parser/3rd_party/LLVM/bin/Linux/libclang.so.12: undefined reference to `exp2@GLIBC_2.29'
../engine/source/meta_parser/3rd_party/LLVM/bin/Linux/libclang.so.12: undefined reference to `log@GLIBC_2.29'
../engine/source/meta_parser/3rd_party/LLVM/bin/Linux/libclang.so.12: undefined reference to `set_curterm@NCURSES6_TINFO_5.0.19991023'
../engine/source/meta_parser/3rd_party/LLVM/bin/Linux/libclang.so.12: undefined reference to `exp@GLIBC_2.29'
../engine/source/meta_parser/3rd_party/LLVM/bin/Linux/libclang.so.12: undefined reference to `pow@GLIBC_2.29'
collect2: error: ld returned 1 exit status
ninja: build stopped: subcommand failed.

**Desktop | 桌面环境**
 - OS: linux
 - Version ubuntu18.04
 - CPU Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz
 - GPU [AMD 6800 XT, NVidia RTX 2060]
 - Memory [e.g. 16GB]
 - Compiler [e.g. MSVC2019, Clang13, G++7.0]

**Additional context | 补充信息**

**Comment by Ol6rin** [comment]
Sorry that currently we only tested on Ubuntu 20.04. Previous versions of Ubuntu is not guaranteed to be supported.
